test/CMakeLists.txt

```diff
@@ -316,11 +316,12 @@ if (BUILD_PYTHON AND BUILD_TESTS)
     #add_pytest_to_ctest(python_tests_PickleTests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_pickle.py")
     add_pytest_to_ctest(python_tests_Python_api_tests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_python_api.py")
     add_pytest_to_ctest(python_tests_RRTests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_rrtests.py")
-    add_pytest_to_ctest(python_tests_RunStochasticTestSuite "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_runStochasticTestSuite.py")
     add_pytest_to_ctest(python_tests_SettingTests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_settings.py")
     add_pytest_to_ctest(python_tests_SteadyStateSolverTests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_steadyStateSolver.py")
     #add_pytest_to_ctest(python_tests_StructuralAnalysisTests "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_structuralAnalysis.py")
 
+    # add_pytest_to_ctest(python_tests_RunStochasticTestSuite "${RR_PYTHON_TESTING_BUILD_PREFIX}/test_runStochasticTestSuite.py")
+
```

Comment:
This isn't a good sign--these tests easily ran in under an hour before the switch to LLVM 13.  However, there was a single test that was inexplicably slow in this suite that I commented out--maybe with a slightly different execution path, there's a different test or two that is inexplicably slow now?  I'll investigate.

Reply:
https://dev.azure.com/TheRoadrunnerProject/255e6ce3-64bc-4a95-a8c8-2a25dc15a90e/_apis/build/builds/1158/logs/183
took 18 minutes.

https://dev.azure.com/TheRoadrunnerProject/255e6ce3-64bc-4a95-a8c8-2a25dc15a90e/_apis/build/builds/1244/logs/242
took >40 and timed out.

Reply:
I've double checked the [RandomTests](https://github.com/sys-bio/roadrunner/blob/6325f0c9ac09740787ea17625918bbe0455b0eb7/test/llvm_tests/RandomTests.cpp) and they run in an expected amount of time. I'm not sure why some tests hang, I wonder if its a similar problem that we had with the pickle?

Reply:
I ran python_test_runStochasticTests.py just locally (windows) and got this:

```
  File "C:\Miniconda3\envs\py39\lib\unittest\case.py", line 592, in run
    self._callTestMethod(testMethod)
  File "C:\Miniconda3\envs\py39\lib\unittest\case.py", line 550, in _callTestMethod
    method()
  File "D:\roadrunner\roadrunner\test\python\test_runStochasticTestSuite.py", line 322, in test_stoch_00014
    self.runOneTest('00014')
  File "D:\roadrunner\roadrunner\test\python\test_runStochasticTestSuite.py", line 240, in runOneTest
    (nmean_wrong, nsd_wrong, nlnmean_wrong, nlnsd_wrong) = self.simulateFile(file, tnum, expected_results)
  File "D:\roadrunner\roadrunner\test\python\test_runStochasticTestSuite.py", line 189, in simulateFile
    sim = rr.simulate(self.start, self.duration, self.steps+1)
  File "D:\roadrunner\roadrunner\cmake-build-release\lib\site-packages\roadrunner\roadrunner.py", line 4603, in simulate
    result = self._simulate(o)
  File "D:\roadrunner\roadrunner\cmake-build-release\lib\site-packages\roadrunner\roadrunner.py", line 4269, in _simulate
    return _roadrunner.RoadRunner__simulate(self, opt)
KeyboardInterrupt
sys:1: ResourceWarning: unclosed file <_io.TextIOWrapper name='../../test/sbml-test-suite/stochastic/results.tsv' mode='w' encoding='cp1252'>

```

Reply:
Okay, possible fix just pushed -- using `for line in open(fname)` ensures that there is no way to close the file again.

Reply:
I updated your fix with calls to f.close(), and I also commented out the newly-slow test (23).  Fingers crossed...

I've also tried running this using my linux VM, and it's EXTREMELY slow there, both with librr 2.1.3 and with the latest (so at least llvm didn't cause problems there).  It looks like it runs in approximately the same amount of time on Windows in Azure, too, which is nice, though it's still confusing about manylinux, since it worked fine there with llvm6.

Reply:
(Also, the 'open' issue was present before when it ran fine, so I wouldn't think it would be the cause of the problem.)

Reply:
I'm filing this as https://github.com/sys-bio/roadrunner/issues/929 and closing this issue.